### ISSUE TITLE
Remove default config file

### ISSUE THREAD

**Issue by MDunitz** [chanzuckerberg/cellxgene#1880]
## Need
- As a developer I want to be confident that the defaults configuration variables are passed in as expected without unnecessary steps and that it is simple to know which configuration values differ from the default.

## Approach
- Currently we have a default_config.yml file at the root of the repo. The file location is hard-coded and always read in and then any additional config files (if they exist) are read in. 
- The default_config file has also become the documentation site for most (but not all) of the config vars. 
After #1879 is complete we should remove the default_config, setting the variables in the configuration classes to the defaults to simplify the codebase

**Comment by MDunitz** [comment]
low priority -> closing
